.github/workflows/man_test.yml

```
@@ -1,3 +1,4 @@
+# This is my workflow name.
```

```suggestion
#This is my workflow name.
```
I would suggest to remove spaces here.